bin/get_schemas.py

```diff
@@ -0,0 +1,106 @@
+import argparse
+import requests
+import configparser
+import xml.etree.ElementTree as ET
+from pathlib import Path
+
+# prsvtoken.py needs to be in the same directory for this to work
+import prsvtoken
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        '--instance',
+        '-i',
+        type=str,
+        required=True,
+        choices=['test', 'prod'],
+        help='Which Preservica instance to access'
+    )
+
+    parser.add_argument(
+        '--destination_folder_path',
+        '-dest',
+        type=str,
+        required=False,
+        help='''Optional. Provide an absolute folder path to save the files
+        in the specified folder'''
+    )
+    return parser.parse_args()
+
+def generate_access_token(config_input: str):
+    config = configparser.ConfigParser()
```

Comment:
I don't understand if the `config. ...` lines are necessary. Seems like you read a file and then don't do anything with it